collections/_article/dev-snapshot-godot-4-4-dev-5.md

```diff
@@ -0,0 +1,135 @@
+test this release well so we can be confident with the changes and release 4.3.1 with them as soon as possible.
+
+Keep in mind that while we try to make sure each dev snapshot is stable enough for general testing, this is by
+definition a pre-release piece of software. Be sure to make frequent backups, or use a version control system such as
+Git, to preserve your projects in case of corruption or data loss.
+
+[Jump to the **Downloads** section](#downloads), and give it a spin right now, or continue reading to learn more about improvements in this release. You can also try the [**Web editor**](https://editor.godotengine.org/releases/4.4.dev5/), [**XR editor**](https://www.meta.com/experiences/godot-game-engine/7713660705416473/), or the **Android editor** for this release (join the [Android editor testing group](https://groups.google.com/g/godot-testers) to get access to pre-release builds).
+
+-----
+
+*The cover illustration is from* [**Unrailed 2: Back on Track**](https://store.steampowered.com/app/2211170/Unrailed_2_Back_on_Track/), *a chaotic co-op railroad construction game where you build your track across procedurally generated worlds! It is developed by [Indoor Astronaut](https://indoorastronaut.ch/). You can purchase the game in early access [on Steam](https://store.steampowered.com/app/2211170/Unrailed_2_Back_on_Track/) and follow the developer on [Twitter](https://x.com/IndoorAstronaut).*
+
+## Highlights
+
+In case you missed them, see the [4.4 dev 1](/article/dev-snapshot-godot-4-4-dev-1/), [dev2](/article/dev-snapshot-godot-4-4-dev-2/),
+[dev 3](/article/dev-snapshot-godot-4-4-dev-3/), and [dev4](/article/dev-snapshot-godot-4-4-dev-4/) release notes for an overview of
+some key features which were already in that snapshot, and are therefore still available for testing in dev 5.
+
+Here are a few highlights that you might find particularly exciting!
+
+### Universalize UID support
+
+Previously, the UID format was limited to only Resource files. This proved to be a pain point for users wishing to reference their scripts and other resources in a manner that's path-agnostic. This, along with a need refactor their scripts anytime these kinds of files were moved, put a significant duty of care on the end-user that shouldn't have been necessary.
+
+Starting with dev5, this will no longer be your burden to bear! Thanks to a long-term effort from [reduz](https://github.com/reduz), UIDs will now be applied universally in a way the engine can automatically track and account for. This is achived via `.uid` files for the resource types that previously didn't support them, functioning similarly to other metadata files that "track" a main file.
+
+(Note that, unlike metadata files, `.uid` files are strictly for the editor; the information is migrated to the uid database on export. For more information, see ([GH-97352](https://github.com/godotengine/godot/pull/97352)).)
```

Comment:
```suggestion
(Note that, unlike metadata files, `.uid` files are strictly for the editor; the information is migrated to the uid database on export. For more information, see [GH-97352](https://github.com/godotengine/godot/pull/97352).)
```

Reply:
Maybe we could tell users to add those files to their versioning software. We tell them that's editor only, but the file still needs to be synced along the team.

Reply:
The syntax for this is actually correct, based on the past articles; I had *everything else* wrong, whoops!

Also good call, added a warning regarding version control